### PR TITLE
cleaning up the automation scripts

### PR DESCRIPTION
**Pull request by MattReimer**
We had some inconsistencies with the shell scripts. I've unified the variable names and also made sure we're not doing dumb things like storing rscontext inside channelarea folder

@KellyMWhitehead could you double check my work on this. So easy to get this stuff wrong

### CODE REVIEW

File: scripts/automation/FargateAllTools.sh [hunk 1]
```diff
@@ -252,7 +256,7 @@ try || {
 try() {
 
   confinement $HUC \
-    $RS_CONTEXT_DIR/hydrology/hydrology.gpkg/network_intersected_300m \
+    $RSCONTEXT_DIR/hydrology/hydrology.gpkg/network_intersected_300m \
     $VBET_DIR/outputs/vbet.gpkg/vbet_50 \
```
Comment (KellyMWhitehead) [inline scripts/automation/FargateAllTools.sh:260]: RVD should be updated to use vbet_68

File: scripts/automation/FargateAllTools.sh [hunk 2]
```diff
@@ -295,14 +299,14 @@ try || {
 try() {
 
   rvd $HUC \
-      $RS_CONTEXT_DIR/hydrology/hydrology.gpkg/network_intersected_300m \
-      $RS_CONTEXT_DIR/vegetation/existing_veg.tif \
-      $RS_CONTEXT_DIR/vegetation/historic_veg.tif \
+      $RSCONTEXT_DIR/hydrology/hydrology.gpkg/network_intersected_300m \
+      $RSCONTEXT_DIR/vegetation/existing_veg.tif \
+      $RSCONTEXT_DIR/vegetation/historic_veg.tif \
       $VBET_DIR/outputs/vbet.gpkg/vbet_50 \
```
Comment (KellyMWhitehead) [inline scripts/automation/FargateAllTools.sh:305]: vbet_68

**Review by KellyMWhitehead — APPROVED** [review]
Looks good overall. I noticed rvd should now be using vbet_68 instead of vbet_50. We can change this now or later when we confirm rvd is working with multi polygon vbets.

**Comment by MattReimer** [comment]
Good catch. I'll make the change. We can always update later if those thresholds change but I want to make sure everything is using the same value for now.

**Comment by MattReimer** [comment]
Note to myself to check and test these automation files

- [ ] FargateAllTools.sh
- [ ] FargateBRAT.sh
- [ ] FargateChannel.sh
- [ ] FargateConfinement.sh
- [ ] FargateRSContext.sh
- [ ] FargateRSContextChannelAreaTauDEM.sh
- [ ] FargateRVD.sh
- [ ] FargateTauDEM.sh
- [ ] FargateVBET.sh

**Comment by joewheaton** [comment]
> Looks good overall. I noticed rvd should now be using vbet_68 instead of vbet_50. We can change this now or later when we confirm rvd is working with multi polygon vbets.

See #415. Philip and I are going to move away from a filename with the threshold as a suffix so it doesn't screw it up if we expose as parameters or change our minds again. FYI

**Pull request merged**